pkg/cluster/samples.go

```diff
@@ -19,17 +20,20 @@ func (m *manager) disableSamples(ctx context.Context) error {
 		return nil
 	}
 
-	return retry.RetryOnConflict(retry.DefaultRetry, func() error {
-		c, err := m.samplescli.SamplesV1().Configs().Get(ctx, "cluster", metav1.GetOptions{})
-		if err != nil {
-			return err
-		}
+	return retry.OnError(retry.DefaultRetry,
+		func(err error) bool {
+			return errors.IsConflict(err) || errors.IsNotFound(err)
+		}, func() error {
```

Comment:
nit: I was confused by this at first glance, can we fix the formatting to have one parameter per-line considering we're doing in-line anonymous functions (that is, huge, formatted parameters)?

```
return retry.OnError(
    a,
    b,
    c
)
```

Reply:
totally, I'll update with re-formatting.